config/default.json

```diff
@@ -1,10 +1,26 @@
 {
-  "ceramic": {
-    "mode": "node",
-    "apiURL": "http://localhost:7007",
-    "ipfsApi": "http://localhost:5011",
-    "anchorServiceAPI": "http://localhost:8081",
-    "ethereumRpc": "http://localhost:7545",
-    "network": "local"
+  "services": {
+    "ceramic": {
+      "mode": "client",
+      "apiURL": "http://localhost:7007",
+      "anchorServiceAPI": "http://localhost:8081",
+      "ethereumRpc": "http://localhost:7545",
+      "network": "dev-unstable"
+    },
+    "ceramic2": {
+      "mode": "client",
+      "apiURL": "http://localhost:7007",
```

Comment:
Shouldn't these use different ports for `apiURL`?

Reply:
sigh yeah probably.  This silly config library means I have to put something here so that the types can be autogenerated, I can't just say "this is a string" and leave its default as undefined.

Reply:
Actually, these "default" configs already don't really make sense.  You probably wouldn't actually use a localhost `ethereumRpc` or `anchorServiceApi` if you're testing on dev-unstable.  But every possible config options needs representation in the default.json...  Maybe I'll change the urls to something totally wrong so its obvious you need to change it.